src/rezplugins/build_system/custom.py

```diff
@@ -200,6 +200,8 @@ def _actions_callback(executor):
                             value = list(map(str, value))
                             value = list(map(quote, value))
                             value = ' '.join(value)
+                        else:
+                            value = quote(str(value))
```

Comment:
This protects against cases where a user passes something like
```
'"`run command`"'
```
We were already ding that for elements in sequences anyway.